### ISSUE TITLE
Add License to Project

### ISSUE THREAD

**Issue by ajubbal** [dukeland/EasyJSWebView#2]
My team and I are interested in using this library but unfortunately cannot until you release the project with an open source license (preferably MIT, BSD, or Apache). Thanks!

**Comment by albertz** [comment]
@dukeland Note that your project is completely useless if you don't say whether we are allowed to use the code or not.

If you want us to freely use it without restriction, say explicitly that the code is **Public Domain**.

If you want to have at least some reference of your name, choose a licence like **zlib**, **simplified BSD** or **MIT**.

**Comment by alex621** [comment]
I am the writer of this class. Dukeland is a brand that no longer exists anymore actually. So use it for whatever and whenever you want. Free for any kind of usage.

**Comment by albertz** [comment]
In that case, you should note somewhere in the Readme or in some Licence.txt that this is Public Domain.

**Comment by alex621** [comment]
Yes. But the problem is that I can't access that account anymore...
